src/uproot/behaviors/TBranch.py

```diff
@@ -892,10 +892,16 @@ def get_from_cache(branchname, interpretation):
             (e, c) for e, c in expression_context if c["is_primary"] and not c["is_cut"]
         ]
 
-        return _ak_add_doc(
+        out = _ak_add_doc(
             library.group(output, expression_context, how), self, ak_add_doc
         )
 
+        if ak_add_doc and type(out).__module__ == "awkward.highlevel":
+            out.layout.parameters["__doc__"] = self.title
+            out.__doc__ = self.title
+
+        return out
```

Comment:
The `_ak_add_doc` function that has just been applied does exactly what these new lines do:

https://github.com/scikit-hep/uproot5/blob/27b3956a431907a3860e6262fa6ecff00b5af75e/src/uproot/behaviors/TBranch.py#L539-L542

except for adding the Python dunder `__doc__`. If that's necessary, we could add it to the `_ak_add_doc` function... but it looks like `ak.Array` already does that, too:

https://github.com/scikit-hep/awkward/blob/4e01cc40e033cb9d06eee869cca73c4be3b0d53c/src/awkward/highlevel.py#L231-L233

In principle, the entire highlighted change should do nothing. Did you see a difference in your output?

Reply:
I'm pretty sure I wasn't seeing the parameter show up correctly until this one. I will check again.